### PR TITLE
Add education landing page

### PR DESCRIPTION
**Pull request by rubenwardy**
The purpose of this page is to encourage the usage of Minetest in education by providing an easy to access guide to the available resources. Eventually, I'd like to have an educational edition of Minetest, which would end up advertised on a section on this page

image

Todo:

- [x] Find good featured projects
- [x] Find better resources
- [x] `rel="nofollow"` ?

### CODE REVIEW

File: _data/edu.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ why:
         more suitable for adoption in organisations where this is a problem.
 
 - title: Friendly Community
-  para: You can find help with any problems on our forums or IRC channels.
+  para: You can find help with any problems on our <a href="http://localhost:4000/get-involved/">forums or IRC channels.</a>
```
Comment (sfan5) [inline _data/edu.yml]: hmmm

Comment (rubenwardy) [reply _data/edu.yml]: Oh rip

**Comment by Calinou** [comment]
Why did you close this PR? Is this page page no longer desired?

**Comment by rubenwardy** [comment]
From the lack of feedback, I didn't think it had much support

**Comment by Calinou** [comment]
I think this is a useful page :slightly_smiling_face: It should be good to merge after conflicts are resolved.

**Comment by rubenwardy** [comment]
Done, the screenshot has also been updated

**Review by Calinou — APPROVED** [review]
Thanks!

I amended the pull request to optimize the images losslessly using `oxipng --strip --zopfli`.

**Pull request merged**